src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -1667,14 +1667,7 @@ skip:
   {
     NOTIFY_NEW_FLUFFY_BLOCK::request fluffy_arg = AUTO_VAL_INIT(fluffy_arg);
     fluffy_arg.current_blockchain_height = arg.current_blockchain_height;    
-    std::vector<blobdata> fluffy_txs;
     fluffy_arg.b = arg.b;
-    fluffy_arg.b.txs = fluffy_txs;
```

Comment:
Removing this line would serialize the entire block entry into the fluffy blob, wouldn't it?

Reply:
Oops, I misread.